pages/team-members/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
+import { gql } from "@apollo/client";
+import client from "../../apollo-client";
```

Comment:
Use named export here

Reply:
What do you mean with named export?

Reply:
Instead of default exporting client from apollo-client.js, you could only export it. Then you import is using { client } here. It is not very important though I think!